repository-data/development/src/main/resources/hcm-content/content/documents/lks/guidance-pages.yaml

```diff
@@ -1127,6 +1127,21 @@
         /hee:detailsContentBlock:
           jcr:primaryType: hippo:mirror
           hippo:docbase: fdf33d93-cb1f-4a12-a954-c5f187e767d2
+      /hee:contentBlocks[16]:
```

Comment:
I'm not sure of this one, as it appears to be referencing button components, however the cms displays three empty contact content blocks:

![image](https://user-images.githubusercontent.com/85481555/151803248-af34b858-ea0a-4ac3-819d-fdc4c785de6b.png)

Having checked the build before this change these contact blocks appear to be populated:

![image](https://user-images.githubusercontent.com/85481555/151803408-6d4be4ff-63a3-47bd-92e9-40993ec1341b.png)

So am not sure, if this shows a breaking change that needs to be resolved manually or via a script, or whether there is something else at play here.

Reply:
It will be resolved via script.